themes/ddbasic/scripts/ddbasic.common.js

```diff
@@ -56,11 +56,25 @@
     });
   }
 
+  /**
+   * HACK: this is an hack that have to be fixed later.
```

Comment:
Please add a comment explaining why this hack is needed. This will help us understand when/how it can be fixed later on.